src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -415,4 +418,20 @@ default boolean damage(double damage, DamageSource damageSource) {
         return getWorld().getEntities(predicate::test);
     }
 
+    /**
+     * Gets the current display name for this {@link Entity}.
+     *
+     * @return The display name
+     */
+    Text getDisplayName();
```

Comment:
Thats the same value I used/would like to use for `TextRepresentable`.
[MixinEntity](https://github.com/ST-DDT/SpongeCommon/commit/5449840f04e65fcc926d152d5f84ed0fff0255a4#diff-811f97d54d477fefea740d226d776a37R175)